### PR TITLE
Validate

### PR DESCRIPTION
**Pull request by GytisLazauskas**

### CODE REVIEW

File: src/app/admin-login/admin-login.component.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { AdminLoginService } from '../services/admin-login-service/admin-login.service';
+
+@Component({
+  selector: 'app-admin-login',
+  templateUrl: './admin-login.component.html',
+  styleUrls: ['./admin-login.component.css']
+})
+export class AdminLoginComponent extends AdminLoginService implements OnInit {
+
+  private adminApiUrl = 'https://good-works-summer-backend.herokuapp.com/initialdata/admin';
```
Comment (PovilasKM) [inline src/app/admin-login/admin-login.component.ts]: Why is Url in a component?

File: src/app/admin-login/admin-login.component.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { Router } from '@angular/router';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { AdminLoginService } from '../services/admin-login-service/admin-login.service';
+
+@Component({
+  selector: 'app-admin-login',
+  templateUrl: './admin-login.component.html',
+  styleUrls: ['./admin-login.component.css']
+})
+export class AdminLoginComponent extends AdminLoginService implements OnInit {
```
Comment (PovilasKM) [inline src/app/admin-login/admin-login.component.ts]: Why is component extending a service?

File: src/app/admin-login/admin-project/admin-project.component.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
```
Comment (PovilasKM) [inline src/app/admin-login/admin-project/admin-project.component.html]: Shouldn't things like popper and query be managed by npm? They should be in dependencies

**Comment by IndirasM** [comment]
Yeah, looks quite problematic. The first major problem with the PR is that you're mixing new code with code formatting, which bloats the PR badly. There could be separate PRs for that.

**Pull request merged**